### PR TITLE
feat: added basic mantine theme

### PR DESCRIPTION
**Pull request by Enubia**

### CODE REVIEW

File: js/react/utils/mantineProviderWrapper.jsx [hunk 1]
```diff
@@ -1,7 +1,81 @@
 import { MantineProvider, createTheme } from '@mantine/core';
 
 const theme = createTheme({
-	/** Put your mantine theme override here */
```
Comment (stacysealky) [inline js/react/utils/mantineProviderWrapper.jsx:4]: Are these the colors that we are already using?

File: js/react/pages/admin/routes/agency/overview.jsx [hunk 2]
```diff
@@ -40,6 +50,12 @@ export default function Overview() {
 		}
 	}, [value]);
 
+	useEffect(() => {
```
Comment (stacysealky) [inline js/react/pages/admin/routes/agency/overview.jsx:53]: Is this how you want to handle frontend navigation from now on? 
I previous used window for this in WishCardCreate.jsx. 
setTimeout(() => window.location.replace('/wishcards/manage'), 1000);

Should we use both or unify the redirect method for code consistency?

If we show a success toast message, we should probably let it display for a second before redirecting the page, right?

**Review by stacysealky — APPROVED** [review]
Approved, but please check my comments :)

File: js/react/utils/mantineProviderWrapper.jsx (hunk 1 above)
Comment (Enubia) [reply js/react/utils/mantineProviderWrapper.jsx:4]: I actually have to check them again, I noticed that the mantine generator does not properly use the input colors, so I'll update them again

File: js/react/pages/admin/routes/agency/overview.jsx (hunk 2 above)
Comment (Enubia) [reply js/react/pages/admin/routes/agency/overview.jsx:53]: We can use either, this is just the recommended way from the react router docs

**Comment by Enubia** [comment]
@stacysealky updated the colors, the darkest version of each of them is the one we use in the design

**Pull request merged**